### PR TITLE
[bitnami/prestashop] Chart standardized

### PR DESCRIPTION
**Pull request by migruiz4**
**Description of the change**

This PR standardizes the bitnami/prestashop chart to be in line with the rest of the catalog.

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Comment by randradas** [comment]
looks like some checks need to be fixed.

File: bitnami/prestashop/templates/pvc.yaml [hunk 1]
```diff
@@ -7,15 +7,27 @@ metadata:
...
   {{- if .Values.persistence.hostPath }}
   storageClassName: ""
   {{- end }}
   accessModes:
+  {{- if not (empty .Values.persistence.accessModes) }}
+  {{- range .Values.persistence.accessModes }}
+    - {{ . | quote }}
+  {{- end }}
+  {{- else }}
     - {{ .Values.persistence.accessMode | quote }}
```
Comment (Mauraza) [inline bitnami/prestashop/templates/pvc.yaml:29]: I think you missing change this line 
```suggestion
    - {{ .Values.persistence.accessModes | quote }}
```

**Review by Mauraza — CHANGES_REQUESTED** [review]
Hi @migruiz4,

Thanks for the PR! Could you check the comment

File: bitnami/prestashop/templates/pvc.yaml (hunk 1 above)
Comment (migruiz4) [reply bitnami/prestashop/templates/pvc.yaml:29]: Thank you for the review! That sentence is intentional for backward-compatibility, similar to what we do with the `service.port` using coalesce.

**Review by Mauraza — APPROVED** [review]
LGTM

**Pull request merged**